engine/spec.py

```diff
@@ -72,8 +72,8 @@ class GameSpec:
     def __init__(
         self,
         world: Dict[str, Any],
-        player_spec: Dict[str, Any],
         sprites: Dict[str, Any],
+        player_spec: Dict[str, Any],
```

Comment:
This is a bit of a spurious change; what's the rationale behind it? Same with the docstring.